content/applications/hr/referrals/reporting.rst

```diff
@@ -0,0 +1,86 @@
+=========
+Reporting
+=========
+
+The reporting function in the **Referrals** app helps recruiters and managers learn where applicants
+are applying from, when referred by a current employee. Additionally, the reporting feature shows
+the number of applicants hired, refused, and still in the recruitment pipeline, for each medium.
+
+.. important::
+   Only users with *Administrator* rights for the **Recruitment** app have access to the reporting
+   feature in the **Referrals** app.
+
+Employees referral analysis report
+==================================
+
+To access the *Employees Referral Analysis* report, navigate to :menuselection:`Referrals app -->
+Reporting`. This loads the :guilabel:`Employees Referral Analysis` report, in a default
+:icon:`fa-bar-chart` :guilabel:`Bar Chart`.
+
+The graph is presented in a :icon:`fa-database` :guilabel:`Stacked` view, with the number of
+referrals on the y-axis, and the source, referred to as the :guilabel:`Medium`, of the applicant
+(e.g.: *Facebook*, *LinkedIn*, *Email*, etc.) on the x-axis. If a medium does **not** appear on the
+report, that indicates there are no referrals from that particular medium.
+
+Referral amounts for all stages are displayed, including :guilabel:`Not Hired` (refused),
+:guilabel:`In Progress`, and :guilabel:`Hired`. The default filter is set to the current month.
+
+Hover over any bar to view a popover containing specific data for that particular bar.
+
+In this view, it is easy to see which :guilabel:`Medium` is the most successful.
+
+.. example::
+   In this example, both :guilabel:`Email` and :guilabel:`LinkedIn` are the mediums with the most
+   referrals, but :guilabel:`Email` has the most referrals that were hired.
+
+.. image:: reporting/employee-report.png
+   :align: center
+   :alt: The default report in the Referrals app.
+
```

Comment:
It seems like this image is intended to be within the example admonition?

Also, this image is pretty illegible. I recommend retaking it 
```suggestion
.. example::
   In this example, both :guilabel:`Email` and :guilabel:`LinkedIn` are the mediums with the most
   referrals, but :guilabel:`Email` has the most referrals that were hired.

   .. image:: reporting/employee-report.png
      :align: center
      :alt: The default report in the Referrals app.
```